gradoop-examples/src/main/java/org/gradoop/benchmark/sna/SocialNetworkAnalyticsExample.java

```diff
@@ -189,7 +190,7 @@ private static String getQuery() {
    * @return used construction pattern
    */
   private static String getConstruction() {
-    return "CONSTRUCT (ca)-[new:hasInterest]->(t)";
+    return "(ca)-[new:hasInterest]->(t)";
```

Comment:
The gdl encoder may not need the Tag "CONSTRUCT", still it should be part of the Cypher string.

Reply:
i removed the tag because it prevent this error message from appearing:
``
line 1:10 mismatched input '(' expecting {'[', '{', ':'}
line 1:37 missing ']' at '<EOF>'
``